src/reporter.ts

```diff
@@ -113,7 +137,8 @@ export class Reporter {
 
   finishSuites(): void {
     this.suiteIds.forEach((suiteId) => {
-      this.client.finishTestItem(suiteId, {});
+      const suiteTestCaseId = this.testCaseIds.get(suiteId);
+      this.client.finishTestItem(suiteId, { testCaseId: suiteTestCaseId });
```

Comment:
`...(suiteTestCaseId  && { testCaseId: suiteTestCaseId  })`
Do not set empty `testCaseId`